Sources/ClientRuntime/EventStream/DefaultMessageDecoderStream.swift

```diff
@@ -11,22 +11,22 @@ extension EventStream {
     public struct DefaultMessageDecoderStream<Event: MessageUnmarshallable>: MessageDecoderStream {
         public typealias Element = Event
 
-        let stream: Stream
+        let stream: ReadableStream
```

Comment:
Not necessary, per se, but is a good practice to constrain a type no more than needed.

The `Stream` protocol is just the combined `ReadableStream` and `WritableStream` protocols.  But this type requires no `WritableStream` functionality, so that protocol is removed.